src/mono/wasi/build/WasiApp.targets

```diff
@@ -185,6 +185,7 @@
       <_WasiClangCommonFlags Include="$(_DefaultWasiClangFlags)" />
       <_WasiClangCommonFlags Include="$(WasiClangFlags)" />
       <_WasiClangCommonFlags Include="--sysroot=&quot;$(WASI_SDK_PATH.Replace('\', '/'))share/wasi-sysroot&quot;" />
+      <_WasiClangCommonFlags Include="--target=wasm32-unknown-wasip2" />
```

Comment:
Meta-note: there are a lot of places where the target triple is duplicated.

Reply:
They don't really share nice place where we could have a constant.

Reply:
and maybe effectively, this is a constant :)